README.md

```diff
@@ -102,6 +104,16 @@ cd build
 cmake .. -G "MinGW Makefiles" -DCMAKE_PREFIX_PATH=C:\Qt\5.9\mingw53_32
 cmake --build .
 ```
+### macOS / OS X
+```
+git clone https://github.com/GENIVI/CANdevStudio.git
+cd CANdevStudio
+git submodule update --init --recursive
+mkdir build
+cd build
+cmake .. -GNinja
```

Comment:
For OSX cmake is a bit more complicated as you need to path to Qt. 
So the line should be cmake .. -GNinja -DCMAKE_PREFIX_PATH=/path/to/Qt/lib/cmake